experimental/plugin_api.md

```diff
@@ -32,10 +31,13 @@ There are three types of files which can be put in the plugin directory.
 * `.spec` files are text files containing a URL, such as `unix:///other.sock`.
 * `.json` files are text files containing a full json specification for the plugin.
 
+UNIX domain socket files must be located under `/run/docker/plugins`, whereas
```

Comment:
@proppy that's define at the bottom of this section, like 42. Is that good enough?